frappe/translate.py

```diff
@@ -1302,3 +1306,8 @@ def get_translated_doctypes():
 		"Property Setter", {"property": "translated_doctype", "value": "1"}, pluck="doc_type"
 	)
 	return unique(dts + custom_dts)
+
+
+# Backward compatibility
+get_full_dict = get_all_translations
+load_lang = get_translations_from_apps
```

Comment:
These were horrible names for what they were doing. No other choice :crossed_fingers: